### PR TITLE
LPS-202379 SXPBlueprint Translations

### PR DESCRIPTION
**Pull request by gustavolimav**
https://liferay.atlassian.net/browse/LPS-202379

Cherry-pick changes from https://github.com/liferay-search/liferay-portal/pull/1177

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
@oliv-yu added one more commit to FE, can you review it for me?

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by oliv-yu** [comment]
@gustavolimav that commit looked good to me, though I'll request @thektan to review the FE now 😅

**Comment by gustavolimav** [comment]
ci:test:sf

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js [hunk 1]
```diff
@@ -232,21 +231,23 @@ function EditSXPBlueprintForm({
...
+				if (Object.hasOwn(responseContent, 'title')) {
+					openErrorToast({
+						message: responseContent.title,
+					});
+				}
+				else if (Object.hasOwn(responseContent, 'errors')) {
+					responseContent.errors.forEach((message) =>
+						openErrorToast({message})
+					);
+				}
```
Comment (thektan) [inline modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js:250]: @oliv-yu We should also add the default error to appear if none of these properties are found.

```js
else {
    throw DEFAULT_ERROR;
}
```

**Review by thektan — CHANGES_REQUESTED** [review]
@oliv-yu Just 1 thing I noticed

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js (hunk 1 above)
Comment (oliv-yu) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js:250]: @thektan Oops, thanks for catching that!! x_x

@gustavolimav could you add in the last commit on https://github.com/oliv-yu/liferay-portal/commits/LPS-202379-error-handling? Thanks!

Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js:250]: @oliv-yu I will take a look at this today

**Pull request closed without merging**